conclave-host/src/main/kotlin/com/r3/conclave/host/internal/attestation/EnclaveQuoteServiceDCAP.kt

```diff
@@ -1,25 +1,23 @@
 package com.r3.conclave.host.internal.attestation
 
 import com.r3.conclave.common.internal.*
-import com.r3.conclave.host.AttestationParameters
 import com.r3.conclave.host.internal.Native
 import com.r3.conclave.host.internal.NativeLoader
 import java.nio.ByteBuffer
 
-class EnclaveQuoteServiceDCAP(val attestationParameters: AttestationParameters.DCAP): EnclaveQuoteService() {
+class EnclaveQuoteServiceDCAP : EnclaveQuoteService() {
 
     override fun initializeQuote(): Cursor<SgxTargetInfo, ByteBuffer> {
         val targetInfo = Cursor.allocate(SgxTargetInfo)
-        Native.initQuoteDCAP(NativeLoader.libsPath.toString(), targetInfo.buffer.array())
+        Native.initQuoteDCAP(NativeLoader.libsPath.toString(), false, targetInfo.buffer.array())
```

Comment:
I think the parameter should be "load", not "skip". I initially though this is skipping the DCAP lib.

Reply:
It is skipping the quoting part of it actually. Would `loadQuotingLibraries` and revert the logic be fine?

Reply:
Yes, I think it's clearer